vendor/github.com/palantir/go-baseapp/baseapp/config.go

```diff
@@ -34,6 +35,19 @@ type HTTPConfig struct {
 	ShutdownWaitTime *time.Duration `yaml:"shutdown_wait_time" json:"shutdownWaitTime"`
 }
 
+// SetValuesFromEnv sets values in the configuration from coresponding
+// environment variables, if they exist. The optional prefix is added to the
+// start of the environment variable names.
+func (c *HTTPConfig) SetValuesFromEnv(prefix string) {
```

Comment:
@jasei any idea why I'm getting [this](https://app.circleci.com/pipelines/github/palantir/policy-bot/514/workflows/21b037a5-7960-4c31-a89c-7f35dc392ccb/jobs/1519)?
```
server/config.go:70:11: c.Server.SetValuesFromEnv undefined (type baseapp.HTTPConfig has no field or method SetValuesFromEnv)
```

Reply:
I believe this is because you modified a vendored file from a library we depend on and part of the build process resets the vendored files to the correct upstream state.

If you want to add this method, you'll have to submit it to https://github.com/palantir/go-baseapp separately and then update that library version here.

Reply:
Now that the `go-baseapp` change merged, you should be able to use it here. After updating the version, be sure to run `./godelw mod` to vendor the new dependency files.